src/commands/org/list/metadata-types.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as fs from 'fs';
+import { Messages } from '@salesforce/core';
+import { DescribeMetadataResult } from 'jsforce/api/metadata';
+import { RegistryAccess } from '@salesforce/source-deploy-retrieve';
+import {
+  Flags,
+  loglevel,
+  orgApiVersionFlagWithDeprecations,
+  requiredOrgFlagWithDeprecations,
+  SfCommand,
+} from '@salesforce/sf-plugins-core';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-org', 'metadata-types');
+
+export class DescribeMetadata extends SfCommand<DescribeMetadataResult> {
+  public static readonly summary = messages.getMessage('description');
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessages('examples');
+  public static readonly aliases = ['force:mdapi:describemetadata'];
+  public static readonly deprecateAliases = true;
+  public static readonly flags = {
+    'api-version': orgApiVersionFlagWithDeprecations,
```

Comment:
The deprecated flag from `sf-plugins-core` doesn't have an alias for `a`:

plugin-source: https://github.com/salesforcecli/plugin-source/blob/main/src/commands/force/mdapi/describemetadata.ts#L27

sf-plugins-core: https://github.com/salesforcecli/sf-plugins-core/blob/main/src/compatibility.ts#L30

Adding it there might cause some conflicts since some commands already use `-a` for other flags:
https://github.com/search?q=org%3Asalesforcecli++%22char%3A+%27a%27%22&type=code